orchagent/intfsorch.cpp

```diff
@@ -64,6 +63,13 @@ void IntfsOrch::doTask(Consumer &consumer)
         string op = kfvOp(t);
         if (op == SET_COMMAND)
         {
+            if (alias == "lo")
+            {
+                addIp2MeRoute(ip_prefix);
+                it = consumer.m_toSync.erase(it);
+                continue;
+            }
+
```

Comment:
suggest move 'alias == lo' block to outside, because it is similar to eth0/docker0. Then 'getPort' can be also outside.